docs/source/conf.py

```diff
@@ -78,6 +78,56 @@
 if RELEASE:
     templates_path = ['_templates-stable'] + templates_path
 
```

Comment:
Probably good to add a comment here with some details on why things are ignored here.

Reply:
adding a TODO comment